apps/nextjs-website/src/lib/fetch/types/LinkTypes.ts

```diff
@@ -0,0 +1,10 @@
+import * as t from 'io-ts';
+
+export const LinkTypesCodec = t.keyof({
```

Comment:
This codec maps the [`LinkType`](https://github.com/pagopa/pagopa-editorial-components/blob/main/src/components/Footer/types.ts#L4) of editorial components which can assume `internal` or `external` values. Where `wrapper` and `social` values come from?

Reply:
`wrapper` and `social` are needed to account for specific cases in the Header's menu and in the Footer's "Follow Us" section respectively

Removing them would mean we would need to create all new LinkTypes just for those specific cases, making the UI on Strapi's side needlessly more complex

Reply:
👍 I've created the task [B2BP-255](https://pagopa.atlassian.net/browse/B2BP-255) to track the activity